ToDoApp/ToDoApp/CreateTaskView+ModelView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CreateTaskView+ModelView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-02.
+//
+
+import Foundation
+import SwiftUI
+
+extension CreateTaskView {
+    @MainActor class CreateTaskViewModel: ObservableObject {
+        @Published private(set) var titleInput: String = ""
+        @Published private(set) var dateInput: Date?
+        @State private var services = FeaturesToDo()
```

Comment:
@State property wrapper is for use in a View, there's no reason to use it in a view model. Also since we are not implementing the business logic yet, let's remove this property completely for now